projects/ng-dynamic-component/src/lib/dynamic.component.ts

```diff
@@ -68,6 +71,7 @@ export class DynamicComponent<C = unknown>
         index: 0,
         injector: this._resolveInjector(),
         projectableNodes: this.ndcDynamicContent,
+        ngModuleRef: this.ndcDynamicModule
```

Comment:
I think it makes sense to also add support for the `environmentInjector` as it's going to replace `ngModuleRef` some time in the future (as per [Angular docs](https://angular.io/api/core/ViewContainerRef#createcomponent): "This [environmentInjector] option is intended to replace the ngModuleRef parameter.")?